content/strategy-goals/strategy/code-graph/search/index.md

```diff
@@ -12,17 +12,19 @@ Quicklinks:
 
 ## Mission
 
-The Search team's mission is to enable developers to find the code they need when they need it.
+The Search team's mission is to enable developers to find the code they need when they need it, and then act on it.
 
 In the near term, we want to make sure our customers are aware of and using our differentiated code search features. For example, Sourcegraph Cloud enables federated search over several code hosts beyond GitHub and GitLab, with relevant results. We envision that Search will power [unified](../index.md#deliver-a-unified-experience) use cases of Sourcegraph, at any scale and across the entire OSS universe.
 
 ## Competitive landscape
 
-Our competitive landscape in the search space is sparse at the moment, but there are serious potential competitors we need to monitor–GitHub is the elephant in the room. It’s very likely they only need to release “good enough” code search to deal Sourcegraph’s search product a serious blow. While we believe our solution will remain far superior to whatever GitHub releases, there is a real risk that the switching costs of using Sourcegraph will be too high for users to switch, if GitHub’s search improves significantly.
+The biggest threat to our dominance as the leader in open-source, free, and commercially available code search is the advnet of GitHub Code search, currently widely available in technology preview. Last quarter we stated:
 
-There are also existing open source projects that may be “good enough” for many users. While we don’t believe these pose a serious threat to our enterprise business, they are competitors with our Cloud offering and, to the extent users prefer those experiences, we need to consider how to compete against them for attention and usage.
+> It’s very likely they only need to release “good enough” code search to deal Sourcegraph’s search product a serious blow. While we believe our solution will remain far superior to whatever GitHub releases, there is a real risk that the switching costs of using Sourcegraph will be too high for users to switch, if GitHub’s search improves significantly.
```

Comment:
Maybe add something like this:

```suggestion
> It’s very likely they only need to release “good enough” code search to deal Sourcegraph’s search product a serious blow. While we believe our solution will remain far superior to whatever GitHub releases, there is a real risk that the switching costs of trying Sourcegraph will be too high if GitHub’s search improves significantly and we don't ensure it's as easy as possible.
```

Reply:
Maybe this section could also be rewritten to be about us providing clear value for money, instead of being about switching costs.